docs/_scripts/notebook_hooks.py

```diff
@@ -54,7 +54,8 @@
     "concepts/index.md": "index.md",
     "concepts/v0-human-in-the-loop.md": "concepts/human-in-the-loop.md",
     "how-tos/index.md": "index.md",
-    "tutorials/introduction.ipynb": "concepts/why-langgraph.md"
+    "tutorials/introduction.ipynb": "concepts/why-langgraph.md",
+    "tutorials/workflows/index.md": "tutorials/workflows/workflows.md",
```

Comment:
Why the change from index.md to workflows.md?

Reply:
messes up the rendering of the section otherwise